.travis.yml

```diff
@@ -7,17 +7,23 @@ script:
 matrix:
   include:
   - osx_image: xcode11.2
-    env: ACTION="swift-package";SDK="iphonesimulator";DESTINATION="platform=iOS Simulator,OS=13.2.2,name=iPad Pro (12.9-inch) (3rd generation)";SHOULD_TEST="true"
+    env: ACTION="swift-package";PLATFORMS="iOS_12,iOS_13";
+
   - osx_image: xcode11.2
-    env: ACTION="swift-package";SDK="appletvsimulator13.2";DESTINATION="platform=tvOS Simulator,name=Apple TV";SHOULD_TEST="true";ENABLE_CODE_COVERAGE="true"
+    env: ACTION="swift-package";PLATFORMS="tvOS_12,tvOS_13";
```

Comment:
In order to not drop code coverage, I need to run tests on multiple platforms before uploading code coverage numbers